src/commands/cmd_server.cc

```diff
@@ -1493,6 +1493,38 @@ class CommandSST : public Commander {
   rocksdb::IngestExternalFileOptions ingest_options_;
 };
 
+// command format: flushmemtables [sync|async]
+class CommandFlushMemTables : public Commander {
+ public:
+  Status Parse([[maybe_unused]] const std::vector<std::string> &args) override {
+    CommandParser parser(args, 1);
+
+    const std::string param = GET_OR_RET(parser.TakeStr());
+    if (util::EqualICase(param, "sync")) {
+      flush_options_.wait = true;
+    } else if (util::EqualICase(param, "async")) {
+      flush_options_.wait = false;
+    } else {
+      return {Status::RedisParseErr, "Parameter must be 'sync' or 'async', got " + param};
+    }
```

Comment:
I think we can have a default behavior `sync`, so it can be `kflush` or `kflush async`, where `sync` is omitted.